frontend/src/components/LoginSignupModal/LoginSignupModal.jsx

```diff
@@ -0,0 +1,143 @@
+import { useState, useEffect } from 'react'
+import { useNavigate } from 'react-router-dom';
+import "./LoginSignupModal.css"
+import baseUrl from '../../../util/baseUrl';
+
+const LoginSignupModal = () => {
+    // For login
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+
+    // For Signup
+    const [signUpPassword, setSignupPassword] = useState("");
+    const [signUpEmail, setSignupEmail] = useState("");
+    const [firstName, setfirstName] = useState("");
+    const [lastName, setlastName] = useState("");
+    const [username, setuserName] = useState("");
+    const [errorMessage, setErrorMessage] = useState('');
+    const [signupStatus, setsignupStatus] = useState('');
+
+    const navigate = useNavigate()
+
+// Sets page title
+useEffect(() => {
+        document.title = 'Login or Sign Up';
+    }, []);
+
+//Logs user in
+const handleLoginSubmit = async (event) => {
+    event.preventDefault();
+
+    let response = await fetch(`${baseUrl}/api/auth/login`, {
+        method: 'POST',
+        headers: {
+        'Content-Type': 'application/json',
+    },
+        body: JSON.stringify({ email: email, password: password })
+    });
+    if (response.status === 200) {
```

Comment:
201 is the standard expected code for a POST request as you have for the other request. Ignore this if that was intentional.